eureka2-ext/eureka2-overrides-aws/src/main/java/com/netflix/eureka2/server/service/overrides/OverridesCodec.java

```diff
@@ -0,0 +1,20 @@
+package com.netflix.eureka2.server.service.overrides;
+
+import com.netflix.eureka2.codec.json.EurekaJsonCodec;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author David Liu
+ */
+public final class OverridesCodec {
```

Comment:
I would keep codecs together in com.netflix.eureka2.codec package, just like metrics are aggregated.